### ISSUE TITLE
Example in README:self.api.active_users():

### ISSUE THREAD

**Issue by guettli** [stphivos/django-mock-queries#137]
I don't understand where `self.api.active_users()` is coming from (at the bottom of the example)

README:

```
""
Function that queries active users
"""
def active_users(self):
    return User.objects.filter(is_active=True).all()

"""
Test function applies expected filters by patching Django's user model Manager or Queryset with a MockSet
"""
from mock import patch
from django_mock_queries.query import MockSet, MockModel


class TestApi(TestCase):
    users = MockSet()
    user_objects = patch('django.contrib.auth.models.User.objects', users)

    @user_objects
    def test_api_active_users_filters_by_is_active_true(self):
        self.users.add(
        	MockModel(mock_name='active user', is_active=True),
        	MockModel(mock_name='inactive user', is_active=False)
        )

        for x in self.api.active_users():
        	assert x.is_active
```

**Comment by stefan6419846** [comment]
This seems to originate from the examples, see https://github.com/stphivos/django-mock-queries/blob/30eb2df6dfe46b772733c196350a9a7f2ec8dbe7/examples/users/analytics/tests.py#L28 Feel free to submit a PR to make this more clear.

**Comment by stefan6419846** [comment]
Fixed in #167.
